TCAT/AppDelegate.swift

```diff
@@ -28,6 +28,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CLLocationManagerDelegate
         
         // Initalize window without storyboard
         let rootVC = HomeViewController()
+        rootVC.getBusStops()
```

Comment:
We've been changing rootVC a lot for testing, is there a better way to make this call that makes it easy to change rootVC?